packaging/aznfs/DEBIAN/postinst

```diff
@@ -72,6 +75,9 @@ chmod 4755 /sbin/mount.aznfs
 mkdir -p /opt/microsoft/aznfs/data
 chmod 0755 /opt/microsoft/aznfs/data
 
+# Create log directory under /etc/stunnel to store stunnel logs
+mkdir -p /etc/stunnel/microsoft/aznfs/nfsv4_fileShare/logs
```

Comment:
Set appropriate permission for this dir as well.

Reply:
Fixed.